src/components/Pages/WithPlayer/widgets/UnitMaterials/Summary/Summary.js

```diff
@@ -24,21 +26,20 @@ const Summary = ({ id }) => {
 
   const summaryLanguages                        = getSummaryLanguages(unit);
   const defaultLanguage                         = selectSuitableLanguage(contentLanguages, summaryLanguages, unit.original_language);
-  const [selectedLanguage, setSelectedLanguage] = useState('');
+  const [selectedLanguage, setSelectedLanguage] = useState(uiLang);
 
   const finalLanguage  = selectedLanguage || defaultLanguage;
   const file           = getFile(unit, finalLanguage);
-  const selectedFileId = file?.id || null;
 
   const handleLanguageChanged = language => {
     setSelectedLanguage(language);
   };
 
   useEffect(() => {
     if (file) {
-      dispatch(assetsActions.doc2html(selectedFileId));
+      dispatch(assetsActions.doc2html(file.id));
```

Comment:
was you check that its not crashed on undefined ?

Reply:
this is fine, it was redundant to define that selectedFileId, probably something from the past